packages/webpack.config.js

```diff
@@ -97,21 +101,36 @@ function getMaterialExternals() {
   return externals;
 }
 
-function getJavaScriptWebpackConfig(entryPath, chunk, modules) {
+const materialExternals = getMaterialExternals();
+
+function getJavaScriptWebpackConfig() {
   return Object.assign(
-    getCommonWebpackParams(entryPath, chunk, {modules}), {
+    getCommonWebpackParams(), {
       externals: Object.assign(
         {
           'react': 'react',
           'classnames': 'classnames',
           'prop-types': 'prop-types',
+          '@material/textfield/constants': `@material/textfield/constants.js`,
         },
-        getMaterialExternals(),
+        materialExternals,
       ),
       module: {
         rules: [{
           test: /\.ts(x)?$/,
           loader: 'ts-loader',
+        }, {
```

Comment:
this was here before, but i realized we need it back, since we do still reference .js files